cfgov/cfgov/settings/base.py

```diff
@@ -77,6 +77,9 @@
     'jobmanager'
 )
 
+if DEPLOY_ENVIRONMENT == 'build':                                                 
+    INSTALLED_APPS += ('eregs_core') 
```

Comment:
I think this needs to be `('eregs_core',)` -- without the trailing comma, this is just a string, not a tuple. So it can't be added to the tuple of apps.